gdb/dwarf2/read.c

```diff
@@ -22098,6 +22292,34 @@ caching, which can slow down startup."),
 			    &set_dwarf_cmdlist,
 			    &show_dwarf_cmdlist);
 
+  add_setshow_enum_cmd ("type-signature-fallback", no_class,
+			type_signature_fallback_enums, &type_signature_fallback,
+			_ ("\
+Set resolution of type signatures with other objfiles as fallback\n\
+(if set before loading symbols)."), _ ("\
+Show resolution of type signatures with other objfiles as fallback\n\
+(if set before loading symbols)."), _ ("\
```

Comment:
It did it like this because there is a similar case where I took this description from, and I tried to stay consistent with existing code. 

https://github.com/dominikmascherbauer/binutils-gdb/blob/532328cf8aff83a80f387acf245cf14effdbccdb/gdb/gdbtypes.c#L6150-L6159

But I guess it would generally make more sense to move that to the detailed description (before explaining the different option enum values) similar to this
https://github.com/dominikmascherbauer/binutils-gdb/blob/532328cf8aff83a80f387acf245cf14effdbccdb/gdb/dwarf2/read.c#L22313-L22319